www/content/scoop.md

```diff
@@ -31,6 +31,9 @@ scoop:
     name: goreleaserbot
     email: goreleaser@carlosbecker.com
 
+  # The project name and current git tag are used in the format string.
+  commit_format: "chore(scoop): update %s version %s"
```

Comment:
This should the templating formatter

`commit_format: "chore(scoop): update {{ . ProjectName }} version {{ . Version }}"`

Reply:
agreed!

Reply:
Updated to template strings, but have a look, just passing the `ctx` context in.